adapter_pipelines/submit.wdl

```diff
@@ -211,6 +213,7 @@ workflow submit {
   Boolean use_caas
   # By default, don't record http requests
   Boolean record_http = false
+  String analysis_file_version = "5.2.1"
```

Comment:
nit: I'm not sure why it's necessary to put a default value here, since submit wdl should never run independently, and each time the metadata version gets updated, we have to change both here and the static input JSON for adapter wdl, even this variable is always overridden by the other value.

Reply:
I added this default here because I was too lazy to update the Optimus and 10x adapter wdls in this PR to pass in the `analysis_file_version`, but I can add those in and remove this default so that's one less place to change the version